pybamm/models/submodels/interface.py

```diff
@@ -19,3 +19,120 @@ def homogeneous_reaction():
         -pybamm.Scalar(1, domain=["positive electrode"]) / pybamm.standard_parameters.lp
     )
     return pybamm.Concatenation(current_neg, current_sep, current_pos)
+
+
+def butler_volmer_lead_acid(c, phi, domain=None):
+    """
+    Butler-Volmer reactions for lead-acid chemistry
+
+    .. math::
+        j = j_0(c) * \\sinh(\\phi - U(c)),
+
+        \\text{where} \\phi = \\Phi_\\text{s} - \\Phi
+
+    Parameters
+    ----------
+    c : :class:`pybamm.Symbol`
+        The electrolyte concentration
+    phi : :class:`pybamm.Symbol`
+        The difference betweent the solid potential and electrolyte potential
+    domain : iterable of strings
+        The domain in which to calculate the interfacial current density. Default is
+        None, in which case the domain is calculated based on c and phi or defaults to
+        the domain spanning the whole cell
+
+    Returns
+    -------
+    :class:`pybamm.Symbol`
+        The interfacial current density in the appropriate domain
+    """
+    if domain is None:
+        # default is for domain to be across the whole cell
+        # (if not specified by c.domain or phi.domain)
+        if c.domain == [] and phi.domain == []:
```

Comment:
I'm thinking in terms of the SPM here so may not be applicable. If the domain=[] then that means that c should be independent of space. But doesn't this returns an interfacial current that will eventually be converted into a vector discretised on the whole_cell?

Reply:
If `c.domain == [] and `phi.domain==[]`, and `domain is None` then we have to pick whether to calculate Butler Volmer in the negative or positive electrode. I think this just makes things confusing though so will raise an error instead if no domain is given by either the string `domain` or `c` or `phi`